entries/c/cisco.com.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Cisco": {
+    "domain": "cisco.com",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
+      "other"
```

Comment:
Maybe this should rather be `cloud` or `remote` (or even both).

Reply:
And IoT, communication, developer, "security", education, etc. I somewhat regret allowing for multiple categories right now :sweat_smile:

Reply:
So what category to include :)